### PR TITLE
Dg add list manual testing

### PR DESCRIPTION
**Pull request by zoranabc201**

### CODE REVIEW

**Review by ngdeqi — COMMENTED** [review]
You can consider the manual test case where there are no clients/properties in the list. Overall, LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -391,42 +395,51 @@ prevent entries retaining within pairingList after it has been deleted from clie
...
   ```
 This block of code is part of ParseManager. It determines the type of list operation(-client, 
--property or -everything) and returns the corresponding object.  
+-property, -pair, or -everything) and returns the corresponding object.  
 Both ParseListClient and ParseListObject then determine if tags are present, and if they are valid, 
 throwing exceptions if any errors are encountered. They then return the corresponding Command type necessary.
 
-* There are five different classes which handle each of the features described above. Each class inherits from the
-   abstract Command class, and reads information present in either the PropertyList or ClientList objects respectively.
+* There are seven different classes which handle each of the features described above. Each class inherits from the
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:427]: For consistency, maybe when you put "7", your subsequent text should also be "7". Would be good if you standardise with one of them.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -499,6 +514,125 @@ java -jar PropertyRentalManager.jar
...
 
 ### List
+1. List clients with or without tags
+   1. Test case `list -client`  
+      Expected: All clients all listed with all their details. The format of details of each client is given below -  
+      ```
+      1. Client Name: Doja Cat
+         Client Contact Number: 93437878
+         Client Email: doja88@example.com
+         Client Budget: 2000
```
Comment (wilsonngja) [inline docs/DeveloperGuide.md:524]: Usually when we print out message for the users, we want it to be clear. That's why when the addition of client is successful, there's a message showing the currency.

Would be highly recommended that all your printing message with regards to money includes a currency.

**Review by wilsonngja — APPROVED** [review]
Overall, LGTM. However, I would highly recommend to include currency in the printing of any money related message.

**Pull request merged**